File: logica-matematica/ejercicio-002/resoluciones/lucas-pajarito/ranking.py
equipos = [
    {
        "nombre": "Rangers",
        "ganados": 8,
        "empatados": 2,
        "perdidos": 2
    },
    {
        "nombre": "Flashes",
        "ganados": 6,
        "empatados": 3,
        "perdidos": 3
    },
    {
        "nombre": "Tigres",
        "ganados": 9,
        "empatados": 1,
        "perdidos": 2
    },
    {
        "nombre": "Leones",
        "ganados": 5,
        "empatados": 4,
        "perdidos": 3
    },
    {
        "nombre": "Panteras",
        "ganados": 4,
        "empatados": 2,
        "perdidos": 6
    }
]


def calcular_puntos(equipos):

    print("=== RANKING DE FÚTBOL SALA ===")

    for equipo in equipos:

        puntos = (equipo["ganados"] * 3) + equipo["empatados"]

        print(f'Equipo: {equipo["nombre"]}')
        print(f'Partidos ganados: {equipo["ganados"]}')
        print(f'Partidos empatados: {equipo["empatados"]}')
        print(f'Partidos perdidos: {equipo["perdidos"]}')
        print(f'Puntos obtenidos: {puntos}')
        print("------------------------------")


calcular_puntos(equipos)

File: logica-matematica/ejercicio-022/resoluciones/lucas-pajarito/ranking.js
const equipos = [
    {
        nombre: "Tigres FC",
        partidos: 20,
        ganados: 15
    },
    {
        nombre: "Leones FS",
        partidos: 20,
        ganados: 12
    },
    {
        nombre: "Águilas United",
        partidos: 18,
        ganados: 14
    },
    {
        nombre: "Halcones FC",
        partidos: 22,
        ganados: 16
    },
    {
        nombre: "Panteras",
        partidos: 19,
        ganados: 10
    }
];

function calcularRendimiento(equipos) {

    console.log("=== RANKING DE FÚTBOL SALA ===");

    equipos.forEach(equipo => {

        const porcentajeVictorias = (equipo.ganados / equipo.partidos) * 100;
        const proporcionVictorias = `${equipo.ganados}:${equipo.partidos}`;

        console.log(`Equipo: ${equipo.nombre}`);
        console.log(`Partidos jugados: ${equipo.partidos}`);
        console.log(`Partidos ganados: ${equipo.ganados}`);
        console.log(`Proporción de victorias: ${proporcionVictorias}`);
        console.log(`Porcentaje de victorias: ${porcentajeVictorias.toFixed(2)}%`);
        console.log("--------------------------------");
    });

}

calcularRendimiento(equipos);